api/app/Policies/PoolPolicy.php

```diff
@@ -21,7 +22,7 @@ class PoolPolicy
      */
     public function viewAny(User $user)
     {
-        return $user->isAdmin();
+        return $user->isAbleTo("view-any-pool");
     }
```

Comment:
this creates problems for anyone with the `pool-operator` role but no `platform-admin`
view any pools belongs only to admins
you cannot query for the pools you have access to via `user.roleAssignments.team.pools` then 

![image](https://user-images.githubusercontent.com/40485260/226732529-ce5ed6bf-17c0-4d12-8641-5bacb02084c6.png)

this breaks the pools page then

![image](https://user-images.githubusercontent.com/40485260/226732750-12199eec-0296-4a9f-b334-81f19a2c4475.png)

Reply:
Good point, this should also be `@can(ability: "view", resolved: true)`